src/v/cluster/partition_balancer_backend.cc

```diff
@@ -273,6 +273,12 @@ void partition_balancer_backend::tick() {
       = ssx::spawn_with_gate_then(
           _gate,
           [this] {
+              if (_tick_in_progress) {
+                  throw balancer_tick_aborted_exception(
+                    "tick already in progress");
```

Comment:
This is going to give a confusing log message right? I wonder if we should just debug log and then return instead.